enterpriseknowledgegraph/entity_reconciliation/get_entity_reconciliation_job_sample_test.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+
+from google.api_core.exceptions import NotFound
+
+import get_entity_reconciliation_job_sample
+
+project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
+location = "global"
+job_id = "5285051433452986163"
+
+
+def test_get_entity_reconciliation_job(capsys):
+    try:
```

Comment:
Same as above.

Reply:
This error handling is because jobs greater than 30 days old are automatically cleared from the history, so this would job id have to be updated every month to stay active.

Reply:
Gotcha. As above, create a new resource during setup, delete in teardown, etc etc.

Reply:
To be addressed in Future PR